### ISSUE TITLE
Support JDK 1.5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Rustemt/metadata-extractor#70]
```
The version requirement was recently lowered from 1.7 to 1.6.

It'd be good to lower it further to 1.5.
```

Original issue reported on code.google.com by `drewnoakes` on 30 Dec 2012 at 10:55

**Comment by GoogleCodeExporter** [comment]
```
After upgrading Adobe's XMPCore to version 5.1.2 I'll be able to lower the JRE 
version requirement to 1.5. This will be part of version 2.7.0.
```

Original comment by `drewnoakes` on 30 Dec 2012 at 10:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This has been completed and will be included in version 2.7.0.
```

Original comment by `drewnoakes` on 31 Dec 2012 at 3:35
- Changed state: **Fixed**
